### DISCUSSION TITLE
How to prohibit user select text?

### DISCUSSION THREAD

**Discussion by mcoo** [wailsapp/wails#2137] [General]
![image](https://user-images.githubusercontent.com/26184088/204149119-e2b4bed9-4cc5-42f6-b2fc-c3791957f56e.png)
This is a button, I don't want user select this text.How to prohibit user select text or image?

**Comment by misitebao** [comment]
You can use the CSS property `user-select:none;`.
https://developer.mozilla.org/en-US/docs/Web/CSS/user-select
